### PR TITLE
Skip malformed fusions (resolves #23)

### LINKED ISSUE #23: Skip malformed fusion products

### PR DESCRIPTION
**Pull request by jpfeil**
@arkal Here is a fix for the empty fusion product. Thanks!

### CODE REVIEW

File: fusion.py [hunk 1]
```diff
@@ -527,6 +527,7 @@ def insert_fusions(transcriptome, fusion_calls, gene_transcripts, peplen, outfil
                                                                                acceptor_name,
                                                                                2 * peplen - 2,
                                                                                len(fusion_peptide)))
+                    break
```
Comment (arkal) [inline fusion.py]: if len(fusion_peptide) < peplen:

**Comment by jpfeil** [comment]
@arkal Ready for review

File: fusion.py (hunk 1 above)
Comment (arkal) [inline fusion.py]: no just less than peplen. not 2*.

Comment (jpfeil) [inline fusion.py]: Thanks for catching that!

**Pull request merged**